x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/package_policies_panel.tsx

```diff
@@ -90,7 +91,7 @@ export const PackagePoliciesPanel = ({ name, version }: PackagePoliciesPanelProp
   const getPackageInstallStatus = useGetPackageInstallStatus();
   const packageInstallStatus = getPackageInstallStatus(name);
   const { pagination, pageSizeOptions, setPagination } = useUrlPagination();
-  const { data } = usePackagePoliciesWithAgentPolicy({
+  const { data, isLoading } = usePackagePoliciesWithAgentPolicy({
```

Comment:
well that's handy

Reply:
😄 
@pzl FYI: Fleet's UI API hooks use kibana's elasticsearch hooks under the hood, which all return and manage a set of attributes - see here https://github.com/elastic/kibana/blob/2e15faeadc64f86a0f94876c9771cdcc15c8840d/src/plugins/es_ui_shared/public/request/use_request.ts#L31-L37